units/crons.go

```diff
@@ -519,3 +519,27 @@ func PopulateHostSetupJobs(env evergreen.Environment, part int) amboy.QueueOpera
 		return catcher.Resolve()
 	}
 }
+
+func PopulateCloudJobs(env evergreen.Environment, part int) amboy.QueueOperation {
```

Comment:
run once a minute? are we sure that the full job easily completes in less than a minute.